docs/rest_api.rst

```diff
@@ -36,37 +40,34 @@ Channel Object
 
 A channel object consists of a
 
-- ``channel_identifier`` should be an ``integer`` containing the identifier of the
+- ``channel_identifier`` should be a string containing the identifier of the
```

Comment:
Since we're describing the schema here (as opposed to user input) it seems a bit odd that every item is described as `should be`. 

I'd just remove all the `should be a / the`